src/impute.ts

```diff
@@ -0,0 +1,53 @@
+import {Encoding} from './encoding';
+import {isFieldDef} from './fielddef';
+import {ImputeTransform} from './transform';
+
+export interface ImputeProperties {
+
+  // Should we add this?
+  // keyChannel?: string;
+
+  /**
+   * The imputation method to use for the field value of imputed data objects.
+   * One of `value`, `mean`, `median`, `max` or `min`.
+   *
+   * __Default value:__  `"value"`
+   */
+  method?: 'value' | 'mean' | 'median' | 'max' | 'min';
```

Comment:
Supported impute method shouldn't be defined separately.  Use defined type from Vega-Util?